.gitignore

```diff
@@ -11,6 +11,7 @@ test/**/test_output
 
 node_modules/
 npm-debug.log
+.yarn-lock
```

Comment:
Just because I installed the devDependencies using yarn its faster, I thought about this already before adding it to this file. Its just a .gitignore file, let it ignore.